admob-latest/build.gradle

```diff
@@ -0,0 +1,64 @@
+apply from: '../config/android/library.gradle'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+apply plugin: 'kotlin-kapt'
+
+android {
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
+    testOptions {
+        unitTests.returnDefaultValues = true
+        unitTests.includeAndroidResources = true
+    }
+}
+
+ext {
+    google_ads_latest = '20.2.0'
+}
+
+dependencies {
+    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+    implementation "com.google.android.gms:play-services-ads:$google_ads_latest"
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$kotlin_coroutines"
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:$kotlin_coroutines"
+    testImplementation "com.google.android.gms:play-services-ads:$google_ads_latest"
+    testImplementation "androidx.test.ext:junit:$androidx_test_ext"
+    testImplementation "androidx.test.ext:junit-ktx:$androidx_test_ext"
+    testImplementation "org.mockito:mockito-android:$mockito"
+    testImplementation "org.mockito:mockito-core:$mockito"
+    testImplementation "org.mockito:mockito-inline:$mockito"
```

Comment:
seems, we don't need testImplementation of `mockito-android`, `mockito-core` & `mockito-inline` for now? 🤔